contract-tests/test_api.py

```diff
@@ -165,3 +165,106 @@ def test_recipe_history(conf, requests_session):
             created = datetime.strptime(revision["date_created"], "%Y-%m-%dT%H:%M:%S.%fZ")
             assert created < last_date
             last_date = created
+
+
+def test_docs(conf, requests_session):
+    r = requests_session.get(conf.getoption("server") + "/api/docs")
+    r.raise_for_status()
+    data = r.json()
+    assert data["_type"] == "document"
+    assert "_meta" in data
+
+
+def test_approval_request(conf, requests_session):
+    r = requests_session.get(conf.getoption("server") + "/api/v1/approval_request")
+    r.raise_for_status()
+    data = r.json()
+    sample_record = data[0]
+    expected_fields = ["approved", "comment", "created", "creator", "id"]
+    expected_approver_fields = ["email", "first_name", "id", "last_name"]
+    expected_creator_fields = ["email", "first_name", "id", "last_name"]
+    assert r.status_code in [200, 302]
```

Comment:
I don't think allowing 30x requests is a good idea. If you include the trailing slashes on the URLs, then this should only return 200.